week9/src/Home.js

```diff
@@ -70,10 +70,15 @@ function saveUserSearch(){
     if(title===''){
         return;
     }else{
-        searchHistory.push(title);
+        searchHistory.push(title)
+        localStorage.setItem('searchHistory', JSON.stringify([searchHistory]));
     }
 }
+const storedSearchHistory = JSON.parse(localStorage.getItem('searchHistory')) || []
 let lastFiveSearches = searchHistory.slice(-5);
```

Comment:
There is nothing restricting this list from growing to hold more than 5 items. I agree you are only pulling the last five with `.slice(-5)` but the list stores all the search history, not just the 5 most recent. You can change this behaviour by popping out elements once we are about to exceed 5 (i.e treating the array like a queue - https://www.geeksforgeeks.org/queue-data-structure/)